week-4/Homework/mandatory/2-writers.js

```diff
@@ -42,3 +42,11 @@ let writers = [
 /*
 If you want an extra challenge, only `console.log()` the writers that are alive.
 */
+
+writers.forEach(writer => {
+  if (writer.alive) {

```

Comment:
you can use the filter array method instead of if

Reply:
Yes, agree with @AlexeyBrambalov try `filter`